Cognite.Simulator.Tests/IntegrationTests/ConnectorRuntimeTest.cs

```diff
@@ -175,7 +175,8 @@ public string GetSimulatorVersion()
             public Task<Dictionary<string, SimulatorValueItem>> RunSimulation(
                 SampleModelFilestate modelState,
                 SimulatorRoutineRevision routineRevision,
-                Dictionary<string, SimulatorValueItem> inputData
+                Dictionary<string, SimulatorValueItem> inputData,
+                CancellationToken _token
```

Comment:
Why do we underscore the parameter name?

Reply:
We usually do it for unused variables, but since our latest changes from master this variable is now in use, so Ive updated it.